packages/sanes-chrome-extension/src/routes/recovery-phrase/index.tsx

```diff
@@ -13,7 +13,10 @@ const RecoveryPhrase = (): JSX.Element => {
   const [mnemonic, setMnemonic] = React.useState<string>('');
 
   React.useEffect((): void => {
-    getMnemonic().then((mnemonic: string): void => setMnemonic(mnemonic));
+    // when this screen is created independent of app (e.g. storybook tests),
+    // persona is undefined
+    const persona = PersonaManager.getWhenCreated();
```

Comment:
As I said before wrap it in a try/catch, the `getWhenCreated` method only adds confusion.